lib/image-diff.js

```diff
@@ -102,6 +104,14 @@ ImageDiff.prototype = {
     var diffPath = options.diffImage;
     var shadow = options.shadow;
 
+    if((typeof options.imageMagick === 'undefined') || (options.imageMagick === true)) {
```

Comment:
As a heads up, we don't need `typeof` to prevent `undefined` errors unless we are checking a global value

```
// Will throw when `window` isn't defined (e.g. in Node.js)
window === 'undefined'

// Won't throw
typeof window === 'undefined'
window.x === undefined
```